### PR TITLE
Use separate buffers for stdout and stderr on executing git-lfs-authenticate

### PR DESCRIPTION
**Pull request by bozaro**
It's fix authentication issue on ssh error messages like:
```
X11 forwarding request failed
```

### CODE REVIEW

**Comment by rubyist** [comment]
This looks reasonable to me :+1:

File: lfs/ssh.go [hunk 1]
```diff
@@ -37,12 +38,22 @@ func sshAuthenticate(endpoint Endpoint, operation, oid string) (sshAuthResponse,
 
 	cmd := exec.Command(exe, args...)
 
-	out, err := cmd.CombinedOutput()
+	// Save stdout and stderr in separate buffers
+	var outbuf, errbuf bytes.Buffer
+	cmd.Stdout = &outbuf
+	cmd.Stderr = &errbuf
```
Comment (sinbad) [inline lfs/ssh.go]: Is that strictly allowed? I thought the only supported way was to read from `cmd.StdErrPipe()` and `cmd.StdOutPipe()`

Comment (bozaro) [inline lfs/ssh.go]: I dont know. This fragment was found on StackOverflow: http://stackoverflow.com/questions/31095699/return-cmd-stdout-and-stderr-as-string-instead-of-printing-to-console-in-golang/31095907
I have no golang experience.

Comment (sinbad) [inline lfs/ssh.go]: Never mind, I see it's OK before Start(). I've just never seen that approach before, useful :)

**Comment by sinbad** [comment]
:+1:

**Comment by technoweenie** [comment]
Thanks!

**Pull request merged**